src/cli/cli.cc

```diff
@@ -1394,6 +1398,21 @@ int main (const int argc, const char* argv[]) {
         log("Could not list devices using " + command);
         exit(1);
       }
+    } else if (targetPlatform == "android") {
+      auto r = exec("adb devices | tail -n +2");
```

Comment:
This expects `adb` to be in `$PATH`. It looks as though you have implemented it correctly below.

Reply:
This change could be covered in Android CI by testing for at least 1 connected emulator.

Reply:
oops good catch!

Reply:
Can come back to that type of test